packages/admin/admin-date-time/src/datePicker/DatePicker.tsx

```diff
@@ -21,20 +22,23 @@ export interface DatePickerProps extends Omit<InputWithPopperProps, "children" |
     maxDate?: Date;
     minDate?: Date;
     slotProps?: SlotProps;
+    startAdornment?: ReactNode;
+    endAdornment?: ReactNode;
```

Comment:
This should be added do the changeset I suppose. Also, aren't these props part of InputWithPopperProps?

Reply:
Yes, they are. I need to pay more attention to the existing props... Changeset will be updated.